src/basics/LocalCache/onlyMineDefaultConfig.ts

```diff
@@ -0,0 +1,47 @@
+export enum OnlyMyFilter {
+  // Test flag is created only for unit-tests
```

Comment:
it doesn't look like it used only in tests, is this comment is still valid or should be removed

Reply:
fixed